modules/ROOT/pages/design-common-problems-raml-10.adoc

```diff
@@ -1026,6 +1065,30 @@ For more information, see the section "Union Types" in the xref:https://github.c
 
 <<Back to the top>>
 
+=== Defining a Weaker Constraint in a Subtype
+
+A subtype that defines a weaker constraint than one defined by its parent is invalid.
```

Comment:
Should we use "child" with "parent"?
Or "subtype" with "primary type"?
parallelism